### PR TITLE
331 | Refactor. Clear Text Traffic set to false

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: packages/openchs-android/android/app/src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
-    <application android:usesCleartextTraffic="true" tools:targetApi="33" tools:ignore="GoogleAppIndexingWarning" />
+    <application android:usesCleartextTraffic="true" tools:targetApi="33" tools:ignore="GoogleAppIndexingWarning" tools:replace="android:usesCleartextTraffic" />
```
Comment (mohan-13) [inline packages/openchs-android/android/app/src/debug/AndroidManifest.xml:7]: Is this change needed here as cleartextTraffic is still allowed on debug mode ?

Comment (SanoferSameera) [reply packages/openchs-android/android/app/src/debug/AndroidManifest.xml:7]: yeah. its `tools:replace="android:usesCleartextTraffic"` to override the usesCleartextTraffic to true irrespective of what’s coming in dependency

Comment (SanoferSameera) [reply packages/openchs-android/android/app/src/debug/AndroidManifest.xml:7]: we have android:usesCleartextTraffic declared inside our abha dependency as well. its giving me error in 
make run if I ignore it. Either I have to remove usesCleartextTraffic from abha or we need to override

**Review by mohan-13 — APPROVED** [review]
